awx/ui_next/CONTRIBUTING.md

```diff
@@ -320,3 +329,9 @@ You can learn more about the ways lingui and its React helpers at [this link](ht
 3) Open up the .po file for the language you want to test and add some translations.  In production we would pass this .po file off to the translation team.
 4) Once you've edited your .po file (or we've gotten a .po file back from the translation team) run `npm run compile-strings`.  This command takes the .po files and turns them into a minified JSON object and can be seen in the `messages.js` file in each locale directory.  These files get loaded at the App root level (see: App.jsx).
 5) Change the language in your browser and reload the page.  You should see your specified translations in place of English strings.
+
+### Marking an issue to be translated
+
+1) Issues marked with `component:I10n` should not be closed after the issue was fixed.
```

Comment:
do you mean component:I18n?

Reply:
I just followed the name of the label that we have on github.


![image](https://user-images.githubusercontent.com/9053044/83155429-3a006400-a0cf-11ea-918f-cd34eb8561d1.png)